index.js

```diff
@@ -10,9 +10,151 @@ function ask(questionText) {
 start();
 
 async function start() {
-  console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
-  let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
-  console.log('You entered: ' + secretNumber);
+  let max = 100;
+  let min = 1;
+
+  let compNum = 50;
+  console.log(
+    "Let's play a game where you (human) make up a number and I (computer) try to guess it."
+  );
+  //compNum = (max + min) / 2;
+  //letting the player set new max in min
+  let newMax = await ask(`what would you like your maximum guess to be?`);
+  max = parseInt(newMax);
+  console.log(max);
+
+  let newMin = await ask(`What would you like you minimum guess to be`);
+  min = parseInt(newMin);
+  console.log(min);
+
+  //setting compNum to the median of the range
+  compNum = Math.floor((max + min) / 2);
+  console.log(compNum);
+
+  let secretNumber = await ask(
+    "What is your secret number?\nI won't peek, I promise...\n"
+  );
+
+  console.log("You entered: " + secretNumber);
   // Now try and complete the program.
-  process.exit();
+  // start with 50(compNum) and ask if it is there number
+  while (true) {
+    let yesOrNo = await ask(`is your number ${compNum}?`);
+    //then run the game function yes or no to move on
+    await gameStart(yesOrNo);
+
+    //process.exit();
+  }
+  function gameStart(yesOrNo) {
+    //run a loop to see if answer is yes or no
+    if (yesOrNo === "yes") {
+      //if yes tell them you found number and end
+      console.log(`i have found you number... ${compNum}!!!`);
+      process.exit();
+      //if yesOrNo is no run next function High or low
+    } else if (yesOrNo === `no`) {
+      isItHighOrLow();
+    }
+  }
+  async function isItHighOrLow() {
+    // ask if it id higher or lower and store as a value
+    while (true) {
+      let highOrLow = await ask(`is this higher(high) or lower(low)`);
+      if (highOrLow === "high") {
+        //if answer is higher run is high function
+        await isHigh();
+      } else if (highOrLow === `low`) {
+        //if answer is lower run isLower function
+        await isLow();
+      }
+    }
+  }
+  async function isHigh() {
```

Comment:
Your isHigh() and isLow() functions work well to narrow down the range and is a good start to the make it smarter story